http2_general/test_h2_specs.py

```diff
@@ -107,10 +108,6 @@ def test_h2_specs(self):
         h2spec.options.extend(
             [
                 "-x generic/2/2",  # Our version TestHalfClosedStreamStateWindowUpdate
-                "-x generic/2/3",  # disabled by issue 1196
-                "-x http2/5.3.1/1",  # disabled by issue 1196
-                "-x http2/5.3.1/2",  # disabled by issue 1196
-                "-x http2/5.1.2",  # disabled by issue 1196
```

Comment:
I'm wondering why `http2_general.test_h2_stream_states.TestHalfClosedStreamStateUnexpectedFrames` isn't unmasked in tests_disabled.json since it's also commented out by https://github.com/tempesta-tech/tempesta/issues/1196

Reply:
Fixed